download/upgradeRecipe/upgradeRecipe8.adoc

```diff
@@ -17,25 +17,25 @@ Any backwards incompatible changes are annotated with a [.label.label-danger.lab
 
 == Backwards incompatible changes to the public API since 7.x
 
-//[.upgrade-recipe-major.upgrade-recipe-public-api]
-//=== Java 11 or higher required
-//
-//If you're using JRE or JDK 8, upgrade to JDK 11 or higher.
-//
-//JDK 11 or higher is still available for _free_ (including security and bug fixes),
-//alongside Oracle's paid subscription.
-//
-//* On linux, get OpenJDK from your linux software repository.
-//For example on Fedora and RHEL:
-//+
-//[source, bash]
-//----
-//sudo dnf install java-11-openjdk-devel
-//----
-//
-//* On Windows and Mac, https://adoptopenjdk.net[download OpenJDK from AdoptOpenJDK].
-//
-//We currently intend to support a minimal version of Java 11 throughout the entire 8.x series.
+[.upgrade-recipe-major.upgrade-recipe-public-api]
+=== Java 11 or higher required
+
+If you're using JRE or JDK 8, upgrade to JDK 11 or higher.
+
+JDK 11 or higher is still available for _free_ (including security and bug fixes),
+alongside Oracle's paid subscription.
+
+* On linux, get OpenJDK from your linux software repository.
+For example on Fedora and RHEL:
++
+[source, bash]
+----
+sudo dnf install java-11-openjdk-devel
+----
+
+* On Windows and Mac, https://adoptopenjdk.net[download OpenJDK from AdoptOpenJDK].
```

Comment:
The most recent version of Apple's operating system for Mac computers is called [macOS](https://en.wikipedia.org/wiki/MacOS).

Reply:
Good point, we have a few other cases of this mistake that we should fix. Look for "Mac\W" regex in find in path.
@rsynek Do you mind taking that along when fixing this one or creating a separate jira for the backlog for it?

Reply:
Fix by an additional commit.